app/models/bulkrax/csv_entry.rb

```diff
@@ -81,7 +87,7 @@ def add_ingested_metadata
     def add_file
       self.parsed_metadata['file'] ||= []
       if record['file']&.is_a?(String)
-        self.parsed_metadata['file'] = record['file'].split(/\s*[;|]\s*/)
+        self.parsed_metadata['file'] = record['file'].split(MULTI_VALUE_SPLIT_DELIMITER_REGEXP)
```

Comment:
looks like this change is related to #429. although eventually we want this value to be set by the user in their "config/bulkrax.rb" file, and then we pull that value to set the split value.

after this pr is merged, can you link it to that issue using the "development" section in either the pr or issue as described [here](https://github.com/samvera-labs/bulkrax/blob/main/CONTRIBUTING.md#submitting-changes)? (linking the issue after the pr is merged won't close the issue)